service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -125,20 +143,56 @@ void shouldReturnErrorsWhenFamilymanNumberIsNotProvided() {
         assertThat(callbackResponse.getErrors()).containsExactly("Enter Familyman case number");
     }
 
-    @Test
-    void shouldTriggerOrderEventWhenSubmitted() throws Exception {
-        String expectedCaseReference = "19898989";
-        postSubmittedEvent(buildCallbackRequest());
+    @TestInstance(PER_CLASS)
+    @Nested
+    class Submitted {
+
+        @AfterEach
+        void resetInvocations() {
+            reset(notificationClient);
+        }
+
+        @Test
+        void submittedShouldNotifyAdminAndLAWhenNoRepresentativesNeedServing() throws Exception {
+            postSubmittedEvent(buildCallbackRequest());
+
+            verify(notificationClient).sendEmail(
```

Comment:
I like this verify eq assertions each on a new line. What do you think?

Reply:
Sure - just saw same thing from 1193 👍